build/scripts/tag-release-candidate.sh

```diff
@@ -63,21 +70,21 @@ git tag --annotate --message "${m}" --force "latest_release"
 git tag --annotate --message "${m}" "${full_version}"
 git push origin "${full_version}"
 
-# Check if we need to bump version in master
-git checkout master
-if [ ! "$(current_branch)" = master ] ; then
-  echo "Must be on master branch"
+# Check if we need to bump version in the default branch
+git checkout "${default_branch}"
+if [ ! "$(current_branch)" = "${default_branch}" ] ; then
+  echo "Must be on ${default_branch} branch"
   exit 7
 fi
-git pull --ff-only origin master
+git pull --ff-only origin "${default_branch}"
 
-master_version=$(release_generate print-version)
+dev_version=$(release_generate print-version)
 
 # Increase next development iteration if needed
-if [ "${master_version}" == "${candidate_for}" ]; then
+if [ "${dev_version}" == "${candidate_for}" ]; then
   echo "Preparing for next development iteration"
   release_generate development
   git add ./pkg/version/release.go
   git commit --message "Prepare for next development iteration"
-  git push origin master:master
+  git push origin "${default_branch}":"${default_branch}"
 fi
```

Comment:
this should work as admins can push (I disabled it when releasing during the week), but we should remember a follow up PR to change this to open a PR instead IMO

Reply:
this may fail actually...

Reply:
@aclevername  see newest commit

Reply:
nice!